docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -32,7 +32,9 @@ If you are not familiar with sysadmin processes and don't need to run checks fro
 === {agent}
 
 beta[] {agent} is a single, unified agent that you can deploy to hosts or containers to collect data and send it to the {stack}.
-Behind the scenes, {agent} runs {beats} or {elastic-endpoint} as required for your configuration. 
+Behind the scenes, {agent} runs {beats} or {elastic-endpoint} as required for your configuration.
```

Comment:
Same comments as above. The `{elastic-endpoint}` is correct, but I would recommend revising the sentence for additional clarity.

Reply:
same response as above. I'm going to use what is becoming our "standard" description of Elastic Agent.